addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -310,6 +310,19 @@ export class Rtc extends Record {
                 this.sfuClient?.disconnect();
             }
         });
+        browser.navigator.serviceWorker?.addEventListener("message", ({ data: { action, id } }) => {
+            if (action === "join_call") {
```

Comment:
It's not obvious where this comes from, I'd suggested making it upper case to better highlight it is an arbitrary constant, or even better actually creating the constant and documenting it should be kept in sync with the python counterpart.